src/marin/download/huggingface/download_hf.py

```diff
@@ -48,88 +48,58 @@ def ensure_fsspec_path_writable(output_path: str) -> None:
 
 @ray.remote
 def stream_file_to_fsspec(cfg: DownloadConfig, hf_fs: HfFileSystem, file_path: str, fsspec_file_path: str):
-    """Ray task to stream a file from HfFileSystem to another fsspec path."""
     target_fs, _ = fsspec.core.url_to_fs(cfg.gcs_output_path)
+    chunk_size = 16 * 1024 * 1024  # 16MB chunks for large files
```

Comment:
When submitting PR, write a high-level comment that describes the change and the motivation